src/blockchain/log-processors/market-migrated.ts

```diff
@@ -3,9 +3,27 @@ import * as Knex from "knex";
 import { FormattedEventLog, ErrorCallback } from "../../types";
 
 export function processMarketMigratedLog(db: Knex, augur: Augur, log: FormattedEventLog, callback: ErrorCallback): void {
-  db.update({ universe: log.newUniverse, needsMigration: db.raw("needsMigration - 1") }).into("markets").where("marketId", log.market).asCallback(callback);
+  db.update({
+    universe: log.newUniverse,
+    needsMigration: db.raw("needsMigration - 1"),
+    needsDisavowal: db.raw("needsDisavowal - 1"),
```

Comment:
Is this necessary for needsDisavowal? I did it for migration, because a market might need to be migrated through more than one fork, can the same thing happen for disavow?

Reply:
I thought this was to support rollback since it is possible the market already needed disavowal from a previous fork and we don't want to set it off on rollback in that case.